src/timed_event.h

```diff
@@ -38,18 +38,21 @@ struct timed_event {
     /** Which faction is responsible for handling this event. */
     int faction_id = -1;
     /** Where the event happens, in global submap coordinates */
```

Comment:
The type `tripoint_abs_ms` says its `map_squares`, not submap coordinates. Maybe the comment needs to be updated?